src/api/window/window.ts

```diff
@@ -681,21 +681,37 @@ export class _Window extends EmitterBase<WindowEvents> {
             .then(({ payload }) => payload.data);
     }
 
+    /*
+     * @deprecated Use {@link Window.disableUserMovement} instead.
+     */
+    public disableFrame(): Promise<void> {
+        console.warn('Function is deprecated; use disableUserMovement instead.');
+        return this.wire.sendAction('disable-window-frame', this.identity).then(() => undefined);
+    }
+
     /**
      * Prevents a user from changing a window's size/position when using the window's frame.
      * @return {Promise.<void>}
-     * @tutorial Window.disableFrame
+     * @tutorial Window.disableUserMovement
      */
-    public disableFrame(): Promise<void> {
+    public disableUserMovement(): Promise<void> {
         return this.wire.sendAction('disable-window-frame', this.identity).then(() => undefined);
     }
 
+    /*
+     * @deprecated Use {@link Window.enableUserMovement} instead.
+     */
+    public enableFrame(): Promise<void> {
```

Comment:
Can we just have this call the other one, not duplicate the call?

Reply:
👍  Looks good otherwise